runironic-inspector.sh

```diff
@@ -1,2 +1,10 @@
 #!/usr/bin/bash
+
+INTERFACE=${INTERFACE:-"provisioning"}
+
+# Allow access to Ironic inspector API
+if ! iptables -C INPUT -i $INTERFACE -p tcp -m tcp --dport 5050 -j ACCEPT > /dev/null 2>&1; then
+    iptables -I INPUT -i $INTERFACE -p tcp -m tcp --dport 5050 -j ACCEPT
```

Comment:
LGTM, though having a comment(_e.g.: "ironic-inspector port"_) makes rule's purpose clearer.

Reply:
Adding a comment is good idea but I think will affect the check at line 6.  Its also inconsistent with how we've been adding iptables rules in other places.